packages/kbn-esql-validation-autocomplete/src/autocomplete/factories.ts

```diff
@@ -199,7 +199,10 @@ export const buildSourcesDefinitions = (
           defaultMessage: `Integration`,
         })
       : i18n.translate('kbn-esql-validation-autocomplete.esql.autocomplete.sourceDefinition', {
-          defaultMessage: `Index`,
+          defaultMessage: '{type}',
+          values: {
+            type: type ?? 'Index',
```

Comment:
Does the tag name 'Alias' need to be internationalized?

Reply:
It is, this is what this function does